### PR TITLE
added: login-signup pages

### PR DESCRIPTION
**Pull request by rkapoor10**
- LOGIN PAGE

login page

- SIGN UP PAGE

signup

### CODE REVIEW

File: login/login.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        <div class="nav-title">
+            <div class="nav-logo">
+                <img src="/assets/rallyalley-logo.png" alt="logo" srcset="" />
+            </div>
+            <div class="flex-title">
+                <div class="fw-bold baseshade-gray">
+                    <a href="/index.html">RALLY<span class="baseteal">ALLEY</span>
+                    </a>
+                </div>
+                <p class="fs-07 baseteal">Smashinn Badminton Products your way</p>
```
Comment (tusharkandpal) [inline login/login.html:27]: if its a typo, correct it `Smashinn`

**Review by tusharkandpal — APPROVED** [review]
Good to merge! 👍🏻

**Review by t007rushi — CHANGES_REQUESTED** [review]
check-box is missing before T&C

File: login/login.html (hunk 1 above)
Comment (rkapoor10) [reply login/login.html:27]: no its not

**Review by t007rushi — APPROVED** [review]

**Pull request merged**